modules/acs-secured-cluster-services-overview.adoc

```diff
@@ -5,18 +5,19 @@
 [id="acs-secured-cluster-services_{context}"]
 = Secured cluster services
 //when updating this page, if applicable, update the corresponding page for ACSCS - acscs-secured-cluster-services-overview.adoc
+//make sure the product name in this file is {product-title} or {product-title-short}.
```

Comment:
Maybe the comment should removed as other writers that might update the file might not understand the meaning here. I do not see conditionals in the file, but I do see two preview links that indicate product naming conventions are correct. 

[1](https://82106--ocpdocs-pr.netlify.app/openshift-acs/latest/architecture/acs-architecture.html#acs-secured-cluster-services_acs-architecture)
[2](https://82106--ocpdocs-pr.netlify.app/openshift-acs/latest/cloud_service/acscs-architecture.html#acscs-secured-cluster-services_acscs-architecture)

Reply:
Unless some top-level attribute is at play here and {product-title-short} can now render as either "RHACS" or "RHACS Cloud Service" depending on the context of a file.